media-gfx/qimgv/qimgv-0.8.8.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+DESCRIPTION="A cross-platform image viewer with webm support, written in qt5"
+HOMEPAGE="https://github.com/easymodo/qimgv"
+SRC_URI="https://github.com/easymodo/qimgv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="exif kde video"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	exif? ( media-gfx/exiv2 )
```

Comment:
package depends on versioned soname of exiv2 so you'll need to add slot op `:=`

Reply:
changed to `media-gfx/exiv2:0`

Reply:
that's a slot dep, not a slot op ;)

https://wiki.gentoo.org/wiki/Sub-slots_and_Slot-Operators

Reply:
Oops xD
I didn't realize packages that depended on exiv2 had to be rebuilt whenever exiv2 updated.
But now grepping at other packages, I see otherwise.